packages/expo-updates/e2e/setup/create-disabled-eas-project.ts

```diff
@@ -31,15 +31,19 @@ const runtimeVersion = '1.0.0';
   const projectRoot = process.env.TEST_PROJECT_ROOT || path.join(workingDir, 'updates-e2e');
   const localCliBin = path.join(repoRoot, 'packages/@expo/cli/build/bin/cli');
 
-  await initAsync(projectRoot, {
-    repoRoot,
-    runtimeVersion,
-    localCliBin,
-    configureE2E: true,
-    transformAppJson: transformAppJsonForUpdatesDisabledE2E,
-    shouldGenerateTestUpdateBundles: false,
-    shouldConfigureCodeSigning: false,
-  });
+  try {
+    await initAsync(projectRoot, {
+      repoRoot,
+      runtimeVersion,
+      localCliBin,
+      configureE2E: true,
+      transformAppJson: transformAppJsonForUpdatesDisabledE2E,
+      shouldGenerateTestUpdateBundles: false,
+      shouldConfigureCodeSigning: false,
+    });
 
-  await setupUpdatesDisabledE2EAppAsync(projectRoot, { localCliBin, repoRoot });
+    await setupUpdatesDisabledE2EAppAsync(projectRoot, { localCliBin, repoRoot });
+  } catch (err) {
+    console.error(err);
+  }
```

Comment:
what was the problem of this?

Reply:
I think it was running yarn install in the created project and there were some issues, and I think this added some more context to the error.

Reply:
just wondering why it's `setupUpdatesDisabledE2EAppAsync` only. but i would lean toward to throw the error and we can catch the problem earlier.

Reply:
I reverted this change to keep diff more minimal.